source/cloud-native-platform/new-component/infrastructure-as-code.html.md.erb

```diff
@@ -1,56 +1,61 @@
 ---
 title: Infrastructure as Code
-last_reviewed_on: 2022-05-17
+last_reviewed_on: 2023-03-15
 review_in: 6 months
 weight: 2
 ---
 
 # <%= current_page.data.title %>
 
-We prefer [Software as a Service](https://en.wikipedia.org/wiki/Software_as_a_service) (SaaS) over [Platform as a Service](https://en.wikipedia.org/wiki/Platform_as_a_service) (PaaS) over [Infrastructure as a Service](https://en.wikipedia.org/wiki/Infrastructure_as_a_service) (IaaS).
+<!-- Lots of acronyms all at once, provide examples/explanation here instead of wikipedia? -->
 
-With this in mind, infrastructure as code refers to
+We prefer [Software as a Service](https://en.wikipedia.org/wiki/Software_as_a_service) (SaaS)
```

Comment:
Is that correct, Saas > PaaS?

Reply:
I'm not sure whether this is eluding to the 'component-level, product-level, platform-level' infrastructure hierarchy or not. But this struck me as odd as well

Reply:
yes SaaS is greater as we don't manage it at all. It's generally harder to acquire here but the statement is right.